gcovr/__main__.py

```diff
@@ -47,10 +47,13 @@
 from .writer.coveralls import print_coveralls_report
 
 
+logger = logging.getLogger('gcovr')
```

Comment:
The issue with that is the configuration of that logger. While I can call the configuration function in every file separately, I don't know how to access the verbose option from the main function. It would either lead to have DEBUG messages always displayed or never. If I do not fix the name, I can't access the logger reliably from the outside.

How to solve the problem?

Reply:
This should be the name of the module and this is fix.

Reply:
I'm sorry, but I'm not sure how to read your reply. What do you suggest exactly?

Reply:
Ok, I got it. In the main script `__name__` isn't `gcovr`. So here it can stay hardcoded but with double quotes.

Reply:
Quotes changed.